libs/remix-ui/editor/src/lib/remix-ui-editor.tsx

```diff
@@ -811,8 +834,13 @@ export const EditorUI = (props: EditorUIProps) => {
         currentFunction.current = functionImpl.name
         executeGptGenerateDocumentationAction.label = intl.formatMessage({id: 'editor.generateDocumentation2'}, {name: functionImpl.name})
         gptGenerateDocumentationAction = editor.addAction(executeGptGenerateDocumentationAction)
-        executegptExplainFunctionAction.label = intl.formatMessage({id: 'editor.explainFunction2'}, {name: functionImpl.name})
+        executegptExplainFunctionAction.label = intl.formatMessage({id: 'editor.explainFunction'}, {name: functionImpl.name})
```

Comment:
This update the label to show only "Explain function". Should this keep using `editor.explainFunction2`?